manifests/plugins_dir.pp

```diff
@@ -14,7 +14,7 @@
       recurse => $recurse,
       purge   => $purge,
       force   => $force,
-      require => Package['sensu'],
+      require => Package[$::sensu::package::package_name],
     }
```

Comment:
Ahhh - this may also explain the above comment.  What components does SE replace?  Is it (essentially) just the dashboard addition to OSS?